src/snowflake/cli/plugins/object/commands.py

```diff
@@ -35,6 +35,29 @@
 ObjectArgument = typer.Argument(
     help="Type of object. For example table, procedure, streamlit.",
     case_sensitive=False,
+    show_default=False,
+)
+# TODO: add documentation link
+ObjectAttributesArgument = typer.Argument(
+    None,
+    help="""Object attributes provided as a list of key=value pairs,
+for example name=my_db 'comment=created with Snowflake CLI'.
```

Comment:
why one has quotes and other does not?

Reply:
If the value contains spaces, it should be quoted. I just checked, `comment='created with Snowflake CLI'` works as well, changed.